src/platform_impl/macos/util/mod.rs

```diff
@@ -131,12 +131,11 @@ pub unsafe fn ns_string_to_rust(ns_string: id) -> String {
   string.to_owned()
 }
 
-#[allow(dead_code)] // In case we want to use this function in the future
+/// Gets the app's name from the `localizedName` property of `NSRunningApplication`
 pub unsafe fn app_name() -> Option<id> {
```

Comment:
I meant you should add another utility function

```rs
unsafe fn app_name_str() -> Option<String> {
  app_name.map(ns_string_to_rust)
}
```

Reply:
Ah, then I misunderstood your comment in Discord. Makes sense though.